modules/katello_content_view.py

```diff
@@ -164,7 +162,8 @@ def main():
     )
 
     if not HAS_NAILGUN_PACKAGE:
-        module.fail_json(msg="Missing required nailgun module (check docs or install with: pip install nailgun")
+        module.fail_json(msg="Missing required nailgun module (check docs or install with: "
+                         "pip install nailgun")
```

Comment:
Some more info, and we could turn Travis on to do this for us to help. Looks like Ansible requires the following to pass:

`pep8 --ignore=E402 --max-line-length=160`

Per http://docs.ansible.com/ansible/dev_guide/developing_modules_in_groups.html